static/sass/components/buttons.scss

```
@@ -50,3 +50,22 @@
 .btn-transparent:active, .btn-transparent:hover {
     color: var(--primary-color)
 }
+
+.sauce-button {
+    background: $bgColorSauceButton;
+    background: -webkit-gradient(linear,right top,left top,from($bgColorSauceButtonGradientFrom),to($bgColorSauceButtonGradientTo));
+    background: linear-gradient(270deg,$bgColorSauceButtonGradientFrom,$bgColorSauceButtonGradientTo);
+    box-shadow: -3px 0 16px -11px #000;
+    border-radius: 290486px;
```

`290486px` seems like a magic number to me, we can also just say:

```suggestion
    border-radius: 2em;
```